apigen.neon

```diff
@@ -1,6 +1,6 @@
 source: ./
 
-destination: docs.woothemes.com/images/sensei-apidocs
+destination: docs.woocommerce.com/images/sensei-apidocs
```

Comment:
👍  Should we wait until this is done before to merge this PR? Everything else seems just fine to me.